hydra_plugins/hydra_wandb_sweeper/wandb_sweeper.py

```diff
@@ -1,174 +1,31 @@
-import functools
-import itertools
-import logging
-import typing
+from typing import List, Optional
 
-import omegaconf
-import wandb
-from hydra.core import plugins
-from hydra.core.override_parser import overrides_parser, types
-from hydra.plugins import sweeper
+from hydra import TaskFunction
+from hydra.plugins.sweeper import Sweeper
+from hydra.types import HydraContext
+from omegaconf import DictConfig
 
-from hydra_plugins.hydra_wandb_sweeper import config
+from hydra_plugins.hydra_wandb_sweeper.config import WandbConfig
 
-SUPPORTED_DISTRIBUTIONS = {
-    "constant": ["value"],
-    "categorical": ["values"],
-    "int_uniform": ["min", "max"],
-    "uniform": ["min", "max"],
-    "q_uniform": ["min", "max"],
-    "log_uniform": ["min", "max"],
-    "q_log_uniform": ["min", "max"],
-    "inv_log_uniform": ["min", "max"],
-    "normal": ["mu", "sigma"],
-    "q_normal": ["mu", "sigma"],
-    "log_normal": ["mu", "sigma"],
-    "q_log_normal": ["mu", "sigma"],
-}
 
+class WandbSweeper(Sweeper):
+    """Class to interface with Wandb"""
 
-def get_wandb_parameter(override):
+    def __init__(self, wandb_sweep_config: WandbConfig, params: Optional[DictConfig]):
+        from hydra_plugins.hydra_wandb_sweeper._impl import WandbSweeperImpl
```

Comment:
Can you please use `importlib.import_module` instead of this?

Reply:
Hahaha yeah I should've done that :) I was semi-blindly following the nevergrad sweeper implementation https://github.com/facebookresearch/hydra/blob/796fdf1c54eabdcd5121d3a9c8ab6af70ee41763/plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/nevergrad_sweeper.py#L16

I agree it's better to use `importlib.import_module` when programatically importing modules.